data/Agda.versions.yml

```diff
@@ -41,7 +41,7 @@ nightly:
           sha256: 19a761f43e0370f8e82e4ea7dd80e4b222e12b9b0a1e1539bf8d6d08d5188e81
           target: windows-2022
   compatibility:
-    agda-stdlib: '1.7.3'
+    agda-stdlib: '1.7.3' - '2.0'
```

Comment:
Oh, that's it!

@patrikja: It should be `'1.7.3 - 2.0'` not `'1.7.3' - '2.0'`.